app/assets/stylesheets/styles/components/guides/_edit_guides.css.scss

```diff
@@ -1,33 +1,33 @@
 .edit-guide-app {
   margin: 2rem 0 4rem;
 
-  small.error{
+  small.error {
     margin-bottom: 0;
   }
 
-  h1{
+  h1 {
     cursor: pointer;
   }
 
-  h1.has-tip, h1.has-tip:hover{
+  h1.has-tip, h1.has-tip:hover {
     border-bottom: none;
     color: inherit;
   }
 
-  .crop-image{
+  .crop-image {
     cursor: pointer;
     color: $secondary-color;
 
-    &.ng-invalid-uploading{
+    &.ng-invalid-uploading {
       p.loading{
         display: block;
       }
-      p.to-upload{
+      p.to-upload {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3